packages/lesswrong/server/vulcan-lib/mutators.ts

```diff
@@ -43,20 +43,80 @@ import { createError } from 'apollo-errors';
 import pickBy from 'lodash/pickBy';
 import { loggerConstructor } from '../../lib/utils/logging';
 
+/**
+ * Note: keep newDocument for backwards compatibility
```

Comment:
According to me [here](https://github.com/ForumMagnum/ForumMagnum/blob/6f6f8fb752a5ace0812d934bf68ff808fbba2add/packages/lesswrong/server/mutationCallbacks.ts#L23), this is backwards. I feel pretty strongly about this, `document` has led to bugs due to misunderstandings.

Reply:
This was just copied over from https://github.com/ForumMagnum/ForumMagnum/blob/master/packages/lesswrong/server/vulcan-lib/mutators.ts#L79; if it's wrong I'll just remove it.